Shell/TheoryFinder.cpp

```diff
@@ -205,12 +205,9 @@ bool TheoryFinder::matchCode(const void* obj,
   unsigned cp = 0; // code pointer
 
   // the clause, if any
-  const Clause* clause;
+  const Clause* clause = nullptr;
   // the length of this clause
-  int clength;
-#ifdef VDEBUG
-  bool clength_assigned = false;
-#endif
```

Comment:
I was able to persuade `clang-tidy` about this in the past using an auxiliary variable. This seems a bit silly in retrospect.